pkg/controllers/provisioning/binpacking/packable.go

```diff
@@ -74,6 +74,9 @@ func PackablesFor(ctx context.Context, instanceTypes []cloudprovider.InstanceTyp
 		packables = append(packables, packable)
 	}
 	// Sort in ascending order so that the packer can short circuit bin-packing for larger instance types
+	// Sort according to the lexicographical order of [CPU, memory]
+	// It may result in cases where an instance type may have larger index when it has larger CPU but smaller memory
```

Comment:
This comment is better placed next to the if check where it applies

Reply:
done